CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 - Generate OLM bundle for Release 23.4.0 ([#270]).
 - Fix LDAP tests for Openshift ([#270]).
+- Missing CRD defaults for `status.conditions` field ([#277]).
```

Comment:
I'd put this under `### Fixed` personally, but whatever. :P

Reply:
How important is that for you :) ?
I mean yeah its a fix, but basically we just added this... i dont mind but i have it under added in every PR now.

Reply:
From a user PoV we haven't added anything new, we've just made sure that an existing (since 23.4.0) feature doesn't break older users.

But no, I wouldn't say it's worth going back and redoing all the existing PRs...